python-flask/README.md

```diff
@@ -0,0 +1,53 @@
+# Python Flask Example
+
+Data consumer example build using Python3 and [Flask](https://github.com/pallets/flask)
+
+## Setup
+
+```console
+$ pip3 install flask
```

Comment:
Best practice should be to install as `--user` or in a virtualenv

```suggestion
$ pip3 install --user flask
```

Reply:
fixed